### PR TITLE
Fix output examples from dsl reference and ctk feature

### PR DESCRIPTION
**Pull request by yzhao244**
**Please specify parts of this PR update:**

- [ ] Specification
- [ ] Schema
- [ ] Examples
- [ ] Extensions
- [ ] Use Cases
- [ ] Community
- [x] CTK
- [x] Other

**Discussion or Issue link**:

#934 

**What this PR does**:

Some fixes to specification version : 1.0.0-alpha

- Fixes output property "as" from two of the examples in the call feature in ctk
- Fixes output property "as" and remove "from" against the example of Output in DSL reference. 

**Additional information:**

Please review in case I misinterpreted any examples of Output in either CTK or DSL reference.

### CODE REVIEW

File: dsl-reference.md [hunk 1]
```diff
@@ -1515,9 +1515,7 @@ schema:
       petId:
         type: string
     required: [ petId ]
-from:
-  petId: '${ .pet.id }'
-to: '.petList += [ . ]'
+as: '.petList += [ . ]'
 ```
```
Comment (JBBianchi) [inline dsl-reference.md:1519]: There is a little mistake here I think. `output.from` became `output.as` and `output.to` became `export.as`.

Comment (yzhao244) [reply dsl-reference.md:1519]: @JBBianchi Thanks for the reply.  Do you mean including both Output and Export properties in this example? Sorry, I feel like I misinterpreted something here lol.. Please let me know  

```
output:
  schema:
    format: json
    document:
      type: object
      properties:
        petId:
          type: string
      required: [ petId ]
  as:                        
    petId: '${ .pet.id }'   
export:
  as: 
   '.petList += [ . ]'  
```

Comment (JBBianchi) [reply dsl-reference.md:1519]: Either that or just the `output` part without `export` but I don't think it hurts to have both.
Thanks a lot ❤️

Comment (yzhao244) [reply dsl-reference.md]: Got it... just updated the example with both Output and Export properties.  :)

**Review by cdavernas — APPROVED** [review]
LGTM! Thanks @yzhao244 ❤️

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**